### PR TITLE
UI: User configurable model path

### PR DESCRIPTION
**Pull request by eranra**

### CODE REVIEW

File: ui/config.yaml [hunk 1]
```diff
@@ -1,18 +1,19 @@
 loglevel: DEBUG
 job:
+  base_path: "../examples/MLCommons"
```
Comment (rachelt44) [inline ui/config.yaml:3]: base path needs to be renamed to input path

File: ui/ui/config_ui.py [hunk 2]
```diff
@@ -18,9 +19,31 @@ def show_page_content(self):
...
+            st.subheader("Configure the base path of the prediction model")
+            st.text_input("Base path", key="base_path", value=get_config("job", "base_path"),
+                          help="The base path of the application prediction model")
+            st.form_submit_button("Save")
+            if st.session_state.base_path != config["job"]["base_path"]:
+                if not os.path.isdir(st.session_state.base_path):
+                    st.error('The provided folder does not exist. Please provide a valid folder path.')
+                    return
+
+                config["job"]["base_path"] = st.session_state.base_path
```
Comment (rachelt44) [inline ui/ui/config_ui.py:35]: base path needs to be renamed to input path

File: ui/ui/config_ui.py [hunk 3]
```diff
@@ -18,9 +19,31 @@ def show_page_content(self):
...
         st.markdown(f"""
                     # ARISE configuration landing page
-                    
-                    ## The UI configuration is:  
                     """)
+        st.divider
+        # allows the user to configure the base path for the prediction model
+        with st.form(key='config-form'):
+            st.subheader("Configure the base path of the prediction model")
+            st.text_input("Base path", key="base_path", value=get_config("job", "base_path"),
```
Comment (rachelt44) [inline ui/ui/config_ui.py:27]: base path needs to be renamed to input path

File: ui/ui/config_ui.py [hunk 4]
```diff
@@ -18,9 +19,31 @@ def show_page_content(self):
...
+                          help="The base path of the application prediction model")
+            st.form_submit_button("Save")
```
Comment (rachelt44) [inline ui/ui/config_ui.py:29]: need to add a browse button so user can select the input path rather than just typing it

**Review by rachelt44 — CHANGES_REQUESTED** [review]

File: ui/config.yaml (hunk 1 above)
Comment (eranra) [reply ui/config.yaml]: Done

File: ui/ui/config_ui.py [hunk 5]
```diff
@@ -18,9 +19,31 @@ def show_page_content(self):
 
         st.markdown(f"""
                     # ARISE configuration landing page
-                    
-                    ## The UI configuration is:  
                     """)
+        st.divider
+        # allows the user to configure the base path for the prediction model
+        with st.form(key='config-form'):
+            st.subheader("Configure the base path of the prediction model")
```
Comment (rachelt44) [inline ui/ui/config_ui.py:26]: please change the title of the page from "Configure the base path of the prediction model" to 
"Configure the input path of the prediction model"

Comment (eranra) [reply ui/ui/config_ui.py:26]: done

File: ui/ui/config_ui.py (hunk 2 above)
Comment (eranra) [reply ui/ui/config_ui.py]: done

File: ui/ui/config_ui.py (hunk 3 above)
Comment (eranra) [reply ui/ui/config_ui.py]: done

File: ui/ui/config_ui.py (hunk 4 above)
Comment (eranra) [reply ui/ui/config_ui.py:29]: Staged to following PR, new issue here: https://github.com/arise-insights/arise-predictions/issues/18

**Pull request merged**